pkg/api/v1/schema.go

```diff
@@ -768,6 +768,15 @@ type Interface struct {
 	// If specified, the virtual network interface will be placed on the guests pci address with the specifed PCI address. For example: 0000:81:01.10
 	// +optional
 	PciAddress string `json:"pciAddress,omitempty"`
+	// Custom DHCP options that may be needed.
+	// +optional
+	DHCPOptions []DHCPOption `json:"dhcpOptions,omitempty"`
```

Comment:
The DHCPOptions seems to be valid only for the `PodNetwork` (bridge and maybe slirp). Would it make sense to move this to the `PodNetwork` struct?